dragon-directory.js

```diff
@@ -123,6 +124,23 @@ function extract_students(record) {
   return students;
 }
 
+// Map grade to value that can be compared and rendered
+function normalize_grade(grade) {
+  const pre_k_grade_value = "6";
```

Comment:
Maybe rename it to normalized_pre_k_grade and then pull it up next to the pre_k_grade constant?

And then add a comment explaining why we're mapping 6 to 0?

Reply:
Added comment - let me rename function too - missed that point when making last commit.

Reply:
Renamed function - if you want me to move render_grade function elsewhere, let me know.